tests/jenkins/install_neuron.sh

```diff
@@ -21,7 +21,7 @@ patch_neuron() (
 set -e
 source ${JENKINS_DIR:-.}/_env_setup.sh
 
-set -x
 patch_neuron
 spack install neuron+debug@develop
+source $SPACK_ROOT/share/spack/setup-env.sh
```

Comment:
This could be removed, right?

Reply:
When there is `no` package installed, `source $SPACK_ROOT/share/spack/setup-env.sh` doesn't do anything.
So we need to re-source after very first package installation to have MODULEPATH updated (and visible for `module av`).